support/cas-server-support-redis-ticket-registry/src/test/java/org/apereo/cas/ticket/registry/RedisPropertyValueTests.java

```diff
@@ -0,0 +1,23 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.configuration.model.support.redis.BaseRedisProperties;
+
+import io.lettuce.core.ReadFrom;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.stream.Stream;
+
+/**
+ * Make sure Redis enumerations are still valid.
+ * @author Hal Deadman
+ * @since 6.4
+ */
+@Tag("Redis")
+public class RedisPropertyValueTests {
```

Comment:
Seems like this test class should go into the redis-core module rather than the registry, or better use, move the tests to the RedisObjectFactoryTests class.